src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -2031,29 +2031,30 @@ public static void townSet(CommandSender sender, String[] split, boolean admin,
 
 	}
 
-	public static void townSetBoard(CommandSender sender, String[] split, Town town, Player player) throws TownyException {
+	public static void townSetBoard(CommandSender sender, String board, Town town) throws TownyException {
 
-		if (split.length < 2)
+		if (board.isEmpty())
 			throw new TownyException("Eg: /town set board " + Translatable.of("town_help_9").forLocale(sender));
 
-		String line = StringMgmt.join(StringMgmt.remFirstArg(split), " ");
-		
-		if (!line.equals("none")) {
-			if (!NameValidation.isValidString(line)) {
+		if ("none".equalsIgnoreCase(board) || "clear".equalsIgnoreCase(board) || "reset".equalsIgnoreCase(board)) {
+			board = TownySettings.getTownDefaultBoard();

```

Comment:
This doesn't allow for a blank board to be set, and will put in the default board. When a blank board is set it is removed from the town status screen, and isn't shown on login, so this should be added back. 

Reset using the default board message is probably fine.

none, clear and reset should be in the tab completes.